data-structures-stacks/insert-under-top.js

```diff
@@ -0,0 +1,16 @@
+/* exported insertUnderTop */
+
+function insertUnderTop(stack, value) {
+  if (stack.peek() === undefined) {
+    return;
+  }
+  const stackValues = [];
+  while (stack.peek() !== undefined) {
```

Comment:
Please solve this without an extra array.

Reply:
Gotcha, arrays have been removed!